tools/experiments/experiments.js

```diff
@@ -143,6 +143,10 @@ const EXPERIMENTS = [
     spec: 'https://github.com/ampproject/amphtml/issues/3991',
     cleanupIssue: 'https://github.com/ampproject/amphtml/issues/4171',
   },
+  {
+    id: 'amp-ios-overflow-x',
+    name: 'Fixes a horizontal scroll issue on iOS browsers.',
```

Comment:
Please add `spec: URL-to-HitGub-issue` and create a bug for cleanupIssue as well.